drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -23,9 +23,9 @@
 from xblock.exceptions import JsonHandlerError
 from xblock.fields import Boolean, Dict, Float, Integer, Scope, String
 from xblock.scorable import ScorableXBlockMixin, Score
+from xblock.utils.resources import ResourceLoader
+from xblock.utils.settings import ThemableXBlockMixin, XBlockWithSettingsMixin
```

Comment:
Yea, I think we should do this for other xblocks as well to maintain compatibility for as long as possible since it's not much more work to do it that way.